content/docs/guides/auth/_index.md

```diff
@@ -24,6 +24,12 @@ The following authentication mechanisms are built-in to gRPC:
   to authenticate the server, and to encrypt all the data exchanged between
   the client and the server. Optional mechanisms are available for clients to
   provide certificates for mutual authentication.
+- **ALTS**: gRPC supports
+  [ALTS](https://cloud.google.com/security/encryption-in-transit/application-layer-transport-security)
+  as a transport security mechanism, if the application is running on
+  [Google Cloud Platform (GCP)](https://cloud.google.com/). See
+  [ALTS Authentication Guide](https://grpc.io/docs/guides/auth/ALTS) for
```

Comment:
The link to the ALTS page should be just `[...](ALTS)`.

Reply:
Oups, my bad: the file name (`ALTS.md`) and the link (`ALTS`) should both be in lower case.